frameworks/person-escort-record/README.md

```diff
@@ -0,0 +1,50 @@
+# Person Escort Record framework
+
+This framework contains the sections, steps and questions that need to be completed to produce
+a Person Escort Record (PER) when moving somebody using the Book a secure move service.
+
+The framework contains a set of YAML defintions that will be used by the [Book a secure move API](https://github.com/ministryofjustice/hmpps-book-secure-move-api)
+and [client applications](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend) of the API to generate validation and form journeys.
+
+## Manifest files
+
+Each manifest file contains a section of the PER. Each manifest contains definitions for the steps and the questions that are required to be answered within a tree-like structure.
+
+Manifest files live in `./frameworks/[framework-name]/manifests/`.
+
+### Manifest options
+
+- `name` **(required)** - name of the section. Used for the step heading caption.
+- `steps` - a list of steps for this section. See [steps documentation](#steps) for options.
```

Comment:
instead of `steps` we could use the same logic as the [digital market place](https://github.com/alphagov/digitalmarketplace-frameworks/blob/c72de5a3c2b2c41c8e3e7a8267c0afafec25d98c/frameworks/digital-outcomes-and-specialists-2/manifests/edit_brief.yml#L4) around `step`, where the highest level of section is the name of the file or some other variable let's say `section` we can add, then we define the category etc. example for risk section `risk.yml`: 

```
name: Hold separately
step: 2
questions:
 normal-question
 discrimination-against-others
```

discrimination-against-others is a multi-question, and in services questions will look like so:
```
name: Discrimination against others

type: multi-question
questions:
  - some-question
  - some-other-question
```

Reply:
As discussed over a call, I think this approach might bloat our questions folder quite a lot as we have very few steps that contain only one question so for each of our steps, we'll need to create a new question type that contains other questions.

More selfishly, it's a bit easier for the frontend to consume step/question breakdowns in this format as it's very similar to how the [form wizard](https://github.com/UKHomeOffice/passports-form-wizard) approaches steps and questions.

I think you're right that we need to consider how we might support an extra "level" within a step to group sub-categories like Vulnerability within Hold separately so I'll take another look at how we could approach that.